spec/lib/task_helpers/imports/generic_object_definitions_spec.rb

```diff
@@ -18,6 +18,7 @@
           'created'    => :datetime,
           'retirement' => :datetime
         },
+        :attribute_constraints => {},
         :associations => {'cloud_tenant' => 'CloudTenant'},
         :methods      => ['kick', 'laugh_at', 'punch', 'parseltongue']
```

Comment:
Going to want to align the `=>` here (assume rubocop will warn about it if it hasn't already)